### DISCUSSION TITLE
Cross-seed

### DISCUSSION THREAD

**Discussion by smark91** [selfhst/icons#173] [New Icon Request]
https://www.cross-seed.org/
https://github.com/cross-seed/cross-seed
https://github.com/cross-seed/cross-seed.org/blob/master/static/img/cross-seed.png
https://github.com/cross-seed/cross-seed.org/blob/master/static/img/cross-seed.svg

**Comment by shollyethan** [comment]
cross-seed has been added - thanks for the links!
